benchmarks/size.js

```diff
@@ -6,7 +6,7 @@ let { bold, gray } = require("../picocolors.js")
 
 async function getJSON(url) {
 	return new Promise(resolve => {
-		get(url, res => {
+		get(url, { headers: {"user-agent": 'picocolors'} },res => {
```

Comment:
```suggestion
		get(url, { headers: {"user-agent": 'picocolors-size-benchmark'} },res => {
```

Reply:
That make sense. Here's commit you ordered! 628580839e388b2b7dbd476561c33ebc165208c5

Reply:
@alexeyraspopov 

Sorry to tag you, but can you check this PR please? Thanks.